clickhouse_sqlalchemy/drivers/compilers/typecompiler.py

```diff
@@ -78,6 +78,9 @@ def visit_datetime64(self, type_, **kw):
     def visit_float32(self, type_, **kw):
         return 'Float32'
 
+    def visit_bfloat32(self, type_, **kw):
```

Comment:
Hi @franz101, think you have a typo here:

```suggestion
    def visit_bfloat16(self, type_, **kw):
```